README.md

```diff
@@ -225,9 +144,7 @@ Example:
 
 > Describe 1 - 3 features you will add to the project.
 
-- [ ] **[new_feature_1]**
-- [ ] **[new_feature_2]**
-- [ ] **[new_feature_3]**
+- No Plans
```

Comment:
[ ]  Kindly I recommend you to update your README file to add  1-3 potential features to be added and please before 😄 .